trunk/cli/images/pgx-builder/Dockerfile

```diff
@@ -1,22 +1,4 @@
-ARG UBUNTU_VERSION=22.04
```

Comment:
the build environment for C extensions seems like it's a subset of the build environment for PGX